wasm/webapi/rejected-arg.any.js

```diff
@@ -1,4 +1,4 @@
-// META: global=window,worker
+// META: global=window,worker,shadowrealm
 
 for (const method of ["compileStreaming", "instantiateStreaming"]) {
```

Comment:
I don't think you should run the webapi tests; these build on `Response` from fetch which I don't think we get in SRs.